Core/Config.cpp

```diff
@@ -1514,16 +1514,12 @@ void Config::RemoveRecent(const std::string &file) {
 
 	private_->ResetRecentIsosThread();
 	std::lock_guard<std::mutex> guard(private_->recentIsosLock);
-	const std::string filename = File::ResolvePath(file);
-	for (auto iter = recentIsos.begin(); iter != recentIsos.end();) {
-		const std::string recent = File::ResolvePath(*iter);
-		if (filename == recent) {
-			// Note that the increment-erase idiom doesn't work with vectors.
-			iter = recentIsos.erase(iter);
-		} else {
-			iter++;
-		}
-	}
+	
+	std::remove_if(recentIsos.begin(), recentIsos.end(), [file](const auto &str) {
+		const auto &filename = File::ResolvePath(file);
```

Comment:
I missed this before, but there's no need to resolve the file path over and over, this can be done outside the loop like before.

Reply:
> I missed this before, but there's no need to resolve the file path over and over, this can be done outside the loop like before.

Yes, you're right, for some reason I stuffed it inside